filesystems/oss/src/main/scala/cromwell/filesystems/oss/batch/OssBatchIoCommand.scala

```diff
@@ -57,7 +57,10 @@ case class OssBatchDeleteCommand(
                                  override val file: OssPath,
                                  override val swallowIOExceptions: Boolean
                                ) extends IoDeleteCommand(file, swallowIOExceptions) with OssBatchIoCommand[Unit, Void] {
-  def operation: Unit = file.ossClient.deleteObject(file.bucket, file.key)
+  def operation: Unit = {
+    file.ossClient.deleteObject(file.bucket, file.key)
```

Comment:
Does this not return `void` anymore? Should we be paying attention to what it does return?

Reply:
> Should we be paying attention to what it does return?

No. It returns a `VoidResult`: https://github.com/aliyun/aliyun-oss-java-sdk/blob/3.13.1/src/main/java/com/aliyun/oss/OSSClient.java#L665